pkg/admission/clusterrtb.go

```diff
@@ -0,0 +1,83 @@
+package admission
```

Comment:
For code structure we might want to move the handlers to separate packages.  All the rbac ones can go in the same pkg, but it seems like we are just going to dump everthing in pkg/admission.  So maybe pkg/cluster and pkg/rbac

Reply:
Done